frontend/src/pages/pipelines/global/modelCustomization/startRunModal/StartRunModal.tsx

```diff
@@ -74,20 +85,37 @@ const StartRunModal: React.FC<StartRunModalProps> = ({ onSubmit, onCancel }) =>
               label="Data science project"
               fieldId="start-run-modal-project-name"
               isRequired
+              labelHelp={
+                <HelperText>
+                  <HelperTextItem>
+                    Select a project for the InstructLab pipeline to run in.
+                  </HelperTextItem>
+                </HelperText>
+              }
             >
               <Stack hasGutter>
                 <StackItem>
-                  <ProjectSelector
-                    isFullWidth
-                    onSelection={(projectName) => {
-                      setSelectedProject(projectName);
-                    }}
-                    namespace={selectedProject ?? ''}
-                    placeholder="Select a Data science project"
-                    isLoading={isLoadingProject}
-                  />
+                  <Flex gap={{ default: 'gapSm' }} direction={{ default: 'column' }}>
```

Comment:
Refer to https://www.patternfly.org/components/forms/form#examples
> When using helper text inside a FormGroup, the HelperText component should be wrapped with the FormHelperText component to provide additional spacing.

Thus, simply wrapping this helper text component to `FormHelperText` should solve your problem, instead of creating a layout by yourself.